android/src/main/java/com/rnimagekeyboard/ReactMediaInputManager.java

```diff
@@ -12,6 +12,7 @@
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.core.os.BuildCompat;
```

Comment:
Please remove `import android.os.Build;` from the imports since it doesn't seem to be used anywhere in this file anymore.